components/Ink/src/MDCInkView.h

```diff
@@ -69,9 +69,8 @@ typedef NS_ENUM(NSInteger, MDCInkStyle) {
 
 /**
  Maximum radius of the ink. If the radius <= 0 then half the length of the diagonal of self.bounds
- is used. This value is ignored if @c inkStyle is set to |MDCInkStyleBounded|.
-
- Ignored if updated ink is used.
+ is used. This value is ignored if @c inkStyle is set to MDCInkStyleBounded and @c
+ usesLegacyInkLayer is set to NO.
```

Comment:
does this mean long term that this property will get removed when usesLegacyInkLayer goes away?

Reply:
If/when `usesLegacyInkLayer` goes away it should say "This value is ignored if `inkStyle` is set to MDCInkStyleBounded."